doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_ironside.rst

```diff
@@ -1012,6 +1014,15 @@ BOOTERROR
   * A non-zero value indicates that an error condition occurred, preventing the CPU from starting.
     Detailed information about the issue can be found in the boot report.
 
+.. _ug_nrf54h20_ironside_se_local_domain_reset:
+
+Local Domain Reset Handling
+****************************
+
+When a local domain resets, |ISE| detects the event in the RESETHUB peripheral and triggers a global system reset, reported as ``SECSREQ`` in the local domain ``RESETINFO.RESETREAS.GLOBAL``.
+This prevents the system from entering an inconsistent state.
```

Comment:
I still think this sentence is not that self-explanatory, and could maybe be rephrased/elaborated a little bit on/maybe just deleted